metrics/cgroups/cgroups.go

```diff
@@ -11,6 +11,7 @@ import (
 	"github.com/containerd/containerd/namespaces"
 	"github.com/containerd/containerd/plugin"
 	"github.com/containerd/containerd/runtime"
+	"github.com/docker/docker/api/errdefs"
```

Comment:
wrong import path

Reply:
i removed the change since we merged stephens PR fixing this